src/sbt-test/sbt-reactive-app/lagom-endpoints/project/plugins.sbt

```diff
@@ -4,5 +4,5 @@ sys.props.get("plugin.version") match {
                          |Specify this property using the scriptedLaunchOpts -D.""".stripMargin)
 }
 
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.3.10")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0")
```

Comment:
@longshorej I'm assuming here that the strategy is to build/test it only of the latest release. I checked the `play-endpoints` equivalent and it's also only using the lastest Play version. 

Shouldn't we have another test for the 1.3.x Lagom series?

Reply:
Just 1.4.0 test is fine. Platform Tooling is only supporting Lagom 1.4 currently, since stable versions of [akka-management](https://github.com/akka/akka-management) (which we use for cluster formation, and also to serve up health/readiness routes) are only published for Akka 2.5.

I don't think this is a problem though - since Platform Tooling is a new project it's okay for us to require the latest Lagom. And since Akka 2.4 is EOL anyways, might make sense for people to get upgrading sooner rather than later.